app/web_hub.go

```diff
@@ -103,6 +156,7 @@ func (a *App) GetHubForUserId(userId string) *Hub {
 	return hub
 }
 
+// HubRegister registers a connection to a hub.
 func (a *App) HubRegister(webConn *WebConn) {
```

Comment:
Should we check if `webConn` is `nil`?

Reply:
Maybe a bit unnecessary. It is only called from api4/websocket and app/websocket_router. Both of them cannot deal with nil connections.

Reply:
Alright. As long as we make sure from the calling side then it's okay.